src/pages/iou/request/step/IOURequestStepAmount.tsx

```diff
@@ -273,25 +278,25 @@ function IOURequestStepAmount({
         }
 
         // If the value hasn't changed, don't request to save changes on the server and just close the modal
-        const transactionCurrency = TransactionUtils.getCurrency(transaction);
-        if (newAmount === TransactionUtils.getAmount(transaction) && currency === transactionCurrency) {
+        const transactionCurrency = TransactionUtils.getCurrency(currentTransaction);
+        if (newAmount === TransactionUtils.getAmount(currentTransaction) && currency === transactionCurrency) {
```

Comment:
Changing how to get the transaction caused this blocker https://github.com/Expensify/App/issues/43259